app/controllers/githook_controller.rb

```diff
@@ -1,11 +1,14 @@
 class GithookController < ApplicationController
+  before_action :login_required
+  before_action :authenticate
+
   def index
     ref = params[:ref]
     success = false
     if ref == "refs/heads/master"
-    	success = system('/var/www/scripts/updatewrapper')
+    	success, _ = pull_master()
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Do not use parentheses for method calls with no arguments.<br>Tab detected.